backend/event/models.py

```diff
@@ -84,6 +84,12 @@ class LetterAction(models.Model):
         blank=True,
     )
 
+    space_descriptions = models.ManyToManyField(
```

Comment:
When we create the actual frontend, we should add a comment here that says: "Pick the most specific description". If I select "St. Radegund's Desk", that also implies that I mean "St. Radegund's Cell" and "Monastery of Holy Cross" and "Poitiers". Else users might feel obliged to add all of these as separate space_descriptions.

Reply:
True. Though if a source says, "Radegund wrote the letter at her desk in her cell in the Monastery of the Holy Cross", I think it's appropriate to select all three.

On the other hadn, selecting only the desk would be appropriate if the source only says "Radegund wrote the letter at her desk", while the connections between Structures show that we know the location of her desk from other sources.